README.md

```diff
@@ -7,17 +7,17 @@ These tilesets display an "X-ray" visualization of Overture data, for inspecting
 Each Overture **theme** has an associated [PMTiles](https://github.com/protomaps/PMTiles) file.
 
 View these tilesets in your browser:
-* [base](https://pmtiles.io/?url=https%3A%2F%2Fhellocdkstack-overturetilesbucket6f38c611-6zusoghoh4au.s3.us-west-2.amazonaws.com%2F2024-06-13-beta%2Fbase.pmtiles)
-* [buildings](https://pmtiles.io/?url=https%3A%2F%2Fhellocdkstack-overturetilesbucket6f38c611-6zusoghoh4au.s3.us-west-2.amazonaws.com%2F2024-06-13-beta%2Fbuildings.pmtiles)
-* [divisions](https://pmtiles.io/?url=https%3A%2F%2Fhellocdkstack-overturetilesbucket6f38c611-6zusoghoh4au.s3.us-west-2.amazonaws.com%2F2024-06-13-beta%2Fdivisions.pmtiles)
-* [places](https://pmtiles.io/?url=https%3A%2F%2Fhellocdkstack-overturetilesbucket6f38c611-6zusoghoh4au.s3.us-west-2.amazonaws.com%2F2024-06-13-beta%2Fplaces.pmtiles)
-* [transportation](https://pmtiles.io/?url=https%3A%2F%2Fhellocdkstack-overturetilesbucket6f38c611-6zusoghoh4au.s3.us-west-2.amazonaws.com%2F2024-06-13-beta%2Ftransportation.pmtiles)
+* [base](https://pmtiles.io/?url=https%3A%2F%2Foverturemaps-tiles-us-west-2-beta.s3.amazonaws.com%2F2024-07-22%2Fbase.pmtiles)
+* [buildings](https://pmtiles.io/?url=https%3A%2F%2Foverturemaps-tiles-us-west-2-beta.s3.amazonaws.com%2F2024-07-22%2Fbuildings.pmtiles)
+* [divisions](https://pmtiles.io/?url=https%3A%2F%2Foverturemaps-tiles-us-west-2-beta.s3.amazonaws.com%2F2024-07-22%2Fdivisions.pmtiles)
+* [places](https://pmtiles.io/?url=https%3A%2F%2Foverturemaps-tiles-us-west-2-beta.s3.amazonaws.com%2F2024-07-22%2Fplaces.pmtiles)
+* [transportation](https://pmtiles.io/?url=https%3A%2F%2Foverturemaps-tiles-us-west-2-beta.s3.amazonaws.com%2F2024-07-22%2Ftransportation.pmtiles)
```

Comment:
We should probably add addresses as well while we're at it.

Reply:
added!